internal/evmreader/evmreader.go

```diff
@@ -29,6 +30,7 @@ type (
 	Context              = context.Context
 	Header               = types.Header
 	Subscription         = ethereum.Subscription
+	Epoch                = model.Epoch
 )
```

Comment:
This idea of defining aliases for types to shorten their names is much "not-Go-like".
I would prefer if we kept the Go standard of having types prefixed by their packages.

Reply:
> I would prefer if we kept the Go standard of having types prefixed by their packages. 

The only exception we currently have is the `model` package. We're using ["dot imports"](https://go.dev/ref/spec#Import_declarations) to add its exported names to the local namespace and avoid prefixing everything with `model.`. [Here](https://github.com/cartesi/rollups-node/blob/next/2.0/internal/repository/validator.go#L11), for example.

Reply:
Done!